Lib/test/test_future.py

```diff
@@ -77,10 +77,13 @@ def test_badfuture10(self):
         self.check_syntax_error(cm.exception, "badsyntax_future10", 3)
 
     def test_ensure_flags_dont_clash(self):
+        # test that future flags and compiler flags doesn't clash
+        # obtain future flags (CO_****) from future module
```

Comment:
I like to add a reference to the issue, it helps to understand why the test exists. 

```suggestion
        # bpo-39562: test that future flags and compiler flags doesn't clash
        
        # obtain future flags (CO_***) from the __future__ module
```